openspec/changes/code-review-bug-finding-and-sidecar-venv-fix/specs/contract-runner/spec.md

```
@@ -0,0 +1,61 @@
+## MODIFIED Requirements
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)